Added acronym package, changed \acronym newcommand to \IFacronym

ilfreddy opened the pull request:
The acronym package is quite useful in this context, keeping track automatically of all used acronyms and making sure they are used in the right (long/short) form.

Then I renamed \acronym to \IFacronym to avoid the name clash

(hope I'm doing the correct pull request this time. Not so experienced about it yet)

ilfreddy committed:
Added acronym package and change proposal acronym command definition to avoid name clash

sean-chester commented:
This is great, thanks @ilfreddy ! On branch _acronym_, I have removed the \IFacronym macro and replaced it with the macro you set up in the acronym package. Does that seem appropriate to you?

ilfreddy commented:
On 11/08/16 15:53 , Sean Chester wrote:

> This is great, thanks @ilfreddy https://github.com/ilfreddy ! On 
> branch /acronym/, I have removed the \IFacronym macro and replaced it 
> with the macro you set up in the acronym package. Does that seem 
> appropriate to you?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> https://github.com/sean-chester/H2020-MSCA-IF-2016/pull/2#issuecomment-239167235, 
> or mute the thread 
> https://github.com/notifications/unsubscribe-auth/ADoaJlKMekTh3_wzJxOgX9jcBAP9sNz2ks5qeylogaJpZM4JhmTY.
> 
> Yes, but I think the acronym of the proposal itself is somehow special, 
> because it is more of a tag rather than a real abbreviation such as IF 
> for Individual Fellowship. It is only used in its abbreviated form and 
> never in the long one. That's why I kept it separate. If you want put it 
> together with the others, I suggest you mark it as "used" at the 
> beginning of the document such that it will always be used in the 
> short-hand form.

Something like:

%put this in the list of acronyms. The user only has to change "TIMPA"
\acro{IFacro}[TIMPA]{This is my proposal acronym}

\begin{document}
%mark IFacro as used, such that the short-hand form is always used.
\acused{IFacro}

Feel free to incorporate these changes if you like them.

Best,
Luca

sean-chester commented:
Nice idea, thanks. I made your suggested change and then merged your pull request.
